dace/codegen/targets/intel_fpga.py

```diff
@@ -176,6 +176,7 @@ def define_stream(self, dtype, buffer_size, var_name, array_size,
         kernel_stream.write("channel {} {}{}{};".format(vec_type, var_name,
                                                         size_str,
                                                         depth_attribute))
+        return 'channel {}'.format(vec_type)
```

Comment:
Both a `kernel_stream.write` and a return in the same function?

Reply:
The return value is for `defined_vars` rather than to generate code. Otherwise we have to copy defined_vars everywhere.